apps/hello-world-vuejs/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "_from": "hello-world-vue",
+  "bundleDependencies": false,
+  "dependencies": {
+    "vue": "^2.5.11",
+    "monday-sdk-js": "^0.0.9"
+  },
+  "deprecated": false,
+  "description": "A Vue.js project",
+  "devDependencies": {
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-env": "^1.6.0",
+    "babel-preset-stage-3": "^6.24.1",
+    "cross-env": "^5.0.5",
+    "css-loader": "^0.28.7",
+    "file-loader": "^1.1.4",
+    "node-sass": "^4.5.3",
+    "sass-loader": "^6.0.6",
+    "vue-loader": "^13.0.5",
+    "vue-template-compiler": "^2.4.4",
+    "webpack": "^3.6.0",
+    "webpack-dev-server": "^2.9.1"
+  },
+  "license": "MIT",
+  "name": "hello-world-vue",
```

Comment:
maybe we should change the names to something like monday.com-hello-world-app-vue